CMPUT301Project/app/src/main/res/layout/addhabits.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this for adding habits or habit events? Rename to add_habit_events if for events